src/webextension/manage/containers/item-filter.js

```diff
@@ -0,0 +1,28 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Localized } from "fluent-react";
+import PropTypes from "prop-types";
+import React from "react";
+import { connect } from "react-redux";
+
+import { filterItems } from "../actions";
+import { parseFilterString } from "../filter";
+import FilterInput from "../../widgets/filter-input";
+
+function ItemFilter({dispatch}) {
+  return (
+    <Localized id="filter-items">
+      <FilterInput placeholder="fILTEr…" onChange={(value) => {
```

Comment:
Also a minor nit worth considering: a placeholder of "Search..." may be more meaningful to the user (Search = what do they want to do, Filter = how that happens). Subject to change either way, I suppose...

Reply:
"Filter" is the word that makes more sense to me... when I see "Search", I usually think "this query will run when I hit enter", and when I see "Filter", I think "this query will run as I type".

Reply:
>when I see "Search", I usually think "this query will run when I hit enter"

That's likely an advanced / technical perspective you're looking at that from but I see your point. 😉 Like I said, minor nit and likely to change again so feel free to ignore me entirely...

Reply:
If it helps, here's some UX discussion about it: https://ux.stackexchange.com/questions/4752/search-vs-filter-what-is-the-difference